inject.js

```diff
@@ -401,7 +401,9 @@
       if (e && !(targetController == controller)) {
         return;
       }
-      showController(controller);
+      if(controller){
```

Comment:
Hmm.. if the controller is not found, shouldn't we simply return and exit out of this function? Why would we allow the rest of the code to run here? Concretely, instead of adding this check, should we add a check in the if above and exit if controller is null?

Reply:
This way allows it to work on websites (like your dailymotion example, but there are many more) that clean the DOM and remove the vsc div but otherwise don't interfere.

Reply:
Torn by this, as it's unexpected and semi broken behavior.

Reply:
I understand your concerns now. Up to you. I personally think it is better than not, since it maintains the core functionality. I don't see it as a hack, but as a method to fail minimally.

If you prefer not let me know (or make the change yourself) and we can just exit. Crashing out is definitely not the best way to fail.